core/node/eth_watch/src/tests/mod.rs

```diff
@@ -0,0 +1,827 @@
+                ),
+            ),
+        ])
+        .await;
+    let chain_log_proofs = chain_log_proofs();
+    sl_client.add_chain_log_proofs(chain_log_proofs).await;
+
+    sl_client.set_last_finalized_block_number(5).await;
+
+    let mut connection = connection_pool.connection().await.unwrap();
+    watcher.loop_iteration(&mut connection).await.unwrap();
+
+    let proof1 = connection
+        .blocks_dal()
+        .get_l1_batch_chain_merkle_path(L1BatchNumber(1))
+        .await
+        .unwrap()
+        .unwrap();
+    let proof1 = hex::encode(&bincode::serialize(&proof1).unwrap());
+    assert_eq!(proof1, "000000000600000000000000420000000000000030783030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303042000000000000003078303030303030303030303030303030303030303030303030303030303030303530303030303030303030303030303030303030303030303030303030303030334200000000000000307830303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030316639420000000000000030783031303230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303042000000000000003078303932343932386331333737613663663234633339633264343666386562396466323365383131623236646333353237653534383339366664346531373362314200000000000000307833373561356266393039636230323134336533363935636136353865303634316537333961613539306630303034646261393335373263343463646239643264");
+
+    sl_client.set_last_finalized_block_number(11).await;
+    watcher.loop_iteration(&mut connection).await.unwrap();
+
+    let proof2 = connection
+        .blocks_dal()
+        .get_l1_batch_chain_merkle_path(L1BatchNumber(2))
+        .await
+        .unwrap()
+        .unwrap();
+    let proof2 = hex::encode(&bincode::serialize(&proof2).unwrap());
+    assert_eq!(proof2, "0100000007000000000000004200000000000000307830303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303031420000000000000030783130613265663736653730396433313862343539626534396631653864376630326437313230663262353031626330616664646439333566316138313363363742000000000000003078303030303030303030303030303030303030303030303030303030303030303930303030303030303030303030303030303030303030303030303030303030334200000000000000307830303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030316639420000000000000030783031303230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303042000000000000003078303932343932386331333737613663663234633339633264343666386562396466323365383131623236646333353237653534383339366664346531373362314200000000000000307861333738613230636132376237616533303731643162643763326164613030343639616263353765343239646436663438613833303932646237303539613138");
```

Comment:
It can be useful for debugging purpose. Imagine you changed something and test failed, you can deserialize string to expected object and compare it with actual